ryan-meinzer.txt

```diff
@@ -0,0 +1,91 @@
+    img and href
+    6. What do we put in the head of our HTML document? 
+    things that we want the browser to know/see but don't want the end user to see
+    7. What is an id?
+    a unique reference for an element (such as a section of a page) 
+    8. What elements can I add an id to? 
+    text
+    9. How many times can I use the same id on a page? 
+    once
+    10. What is a class? 
+    a prototype from which objects are created
+    11. What elements can I add a class to? 
+    objects
+    12. How many times can I use the same class on a page? 
+    infinite
+    13. How do I get my link to open in a new tab?
+    set the target attribute to "_blank"
+    14. What is the alt attribute used for? 
+    alternative text description for an image
+    15. How do I reference an id?
+    #
+    16. What is the difference between a section and a div
+    a div typically holds sections which are areas of a page
+    17. What is CSS used for? 
+    cascading style sheets are used to style/present elements of a page
+    18. How to we select an element? Example - every h2 on the page
+    with it's name "h2"
+    19. What is the difference between a class and an id? - Give me an example of when I might use each one
+    a class is for multiple elements whereas an id is for one; class can be used for multiple paragraphs of text and an id can be used for a specific title or section of a page
+    20. How do we select classes in CSS?
+    with it's name preceeded by a dot ".[name]"
+    21. How do we select a p element with a single class of “human””?
+    .human p
+    22. What is a parent child selector? When would this be useful? 
+    it selects all elements of a child of an element; would be useful for styling an entire section for example
+    23. How do you select all links within a div with the class of sidebar?
+    div.sidebar a
+    24. What is a pseudo selector?
+    selects elements in a specific state
+    25. What do we use the change the spacing between lines?
+    line-height:
+    26. What do we use to change the spacing between letters?
+    letter-spacing:
+    27. What do we use to to change everything to CAPITALS? lowercase? Capitalize?
+    text-transform: uppercase; text-transform: lowercase; text-transform: capitalize
+    28. How do I add a 1px border around my div that is dotted and black?
+    .div {
+        border: 1px dotted #000000
+    }
+    29. How do I select everything on the page? 
+    * 
+    30. How do I write a comment in CSS?
+    /*
+    31. How do I find out what file I am in, when I am using the command line? 
+    pwd
+    32. Using the command line - how do I see a list of files/folders in my current folder?
+    ls
+    33. How do I remove a file via the command line? Why do I have to be careful with this? 
+    rm; because it's irreversible 
+    34. Why should I use version control? 
+    makes it easier to track changes and to collaborate with others on builds with revisions
```

Comment:
I like how you included the collaboration aspect as well

Reply:
coolio